### ISSUE TITLE
Add download counts

### ISSUE THREAD

**Issue by xavivars** [sushain97/apertium-on-github#15]
It'd be great to keep a download_count for each package in the source_browser, either pulling directly the data from the API (maybe on demand so it doesn't use the API too much), or as an offline job that runs every other hour, puts together a json with all the aggregated information, and commits it back to the repo.

**Comment by sushain97** [comment]
Hm, I think we should examine the goal of the source browser. In my eyes,
it's for devs to easily get to packages in a super light/compact interface.
I'm not sure I understand how download count would add to that goal.

If we want to make a more landing page style thing for Apertium on GitHub
as a whole, that's a good (but separate) goal. e.g. a dev doesn't need a
link to the wiki taking up space because they can auto complete it in their
browser faster and they don't need a description of Apertium either but a
landing page should have both those things.

**Comment by xavivars** [comment]
Fair point. The main goal is adressing the concern some people have on losing SF download counts, so let's try to come up with a good solution for that (if we find any: just saw that source code downloads are not counted in any case)

**Comment by sushain97** [comment]
I think a shield would be good:

[![Github All Releases](https://img.shields.io/github/downloads/atom/atom/total.svg)]()

from https://shields.io/ (ctrl+f for "Download").

These could be scripted or manually inserted into repo READMEs.
